packages/amesos2/src/Amesos2_Tacho_decl.hpp

```diff
@@ -96,10 +98,20 @@ class TachoSolver : public SolverCore<Amesos2::TachoSolver, Matrix, Vector>
   typedef Tacho::ordinal_type                                  ordinal_type;
   typedef Tacho::size_type                                        size_type;
   typedef Kokkos::DefaultHostExecutionSpace                   HostSpaceType;
-  typedef Kokkos::TaskScheduler<HostSpaceType>                SchedulerType;
-  typedef Kokkos::View<size_type*,HostSpaceType>            size_type_array;
-  typedef Kokkos::View<ordinal_type*,HostSpaceType>      ordinal_type_array;
-  typedef Kokkos::View<tacho_type*, HostSpaceType>         value_type_array;
+
+  typedef Kokkos::DefaultExecutionSpace                      DeviceSpaceType;
+
+
+  typedef Kokkos::TaskScheduler<DeviceSpaceType>              SchedulerType;
+
```

Comment:
Task scheduler requires execution space as its template parameter.

Reply:
This will currently always be an exec space and naming has been updated.